### PR TITLE
use Into<Value> and Into<Variant> where possible

### PR DESCRIPTION
**Pull request by jf2048**

### CODE REVIEW

File: gtk4/src/expression.rs [hunk 1]
```diff
@@ -313,5 +379,16 @@ macro_rules! define_expression {
                 value
             }
         }
+
+        impl From<$rust_type> for glib::Value {
+            fn from(e: $rust_type) -> Self {
+                skip_assert_initialized!();
+                let mut value = glib::Value::for_value_type::<$rust_type>();
+                unsafe {
+                    ffi::gtk_value_set_expression(value.to_glib_none_mut().0, e.into_glib_ptr())
```
Comment (sdroege) [inline gtk4/src/expression.rs:388]: This will leak the pointer

Comment (jf2048) [reply gtk4/src/expression.rs:388]: Are you sure? This is using `into_glib_ptr` like the rest

Comment (sdroege) [reply gtk4/src/expression.rs:388]: Yes, but it's `set_expression()` and not `take_expression()`

Comment (jf2048) [reply gtk4/src/expression.rs]: Oh right :)

**Review by sdroege — APPROVED** [review]

**Pull request merged**